cfgov/jinja2/v1/_includes/organisms/filter-filterable-list.html

```diff
@@ -0,0 +1,164 @@
+
+<div class="o-filter-filterable-list">
+    {% call expandable.render("Filter", {
+      'is_bordered': true,
+      'is_midtone': true,
+      'is_expanded': true
+    }) %}
+        {{ _filters_form(settings, options) }}
+    {% endcall %}
+</div>
+
+{% endmacro %}
+
+
+{# A helper macro for filter_filterable_list.render()
+   ========================================================================== #}
+{% macro _filters_form(inputs=[], options={}) %}
+{% if options.action %}
+{% set action = options.action if options.action != '/' %}
+{% endif %}
+<form class="form-l"
+      {{ 'method=' ~ options.form_method if options.form_method else 'method=post' }}
+      {{ 'action=' ~ action if action else '' }}>
+    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+
+    {{ _render_form_wrapper(inputs) }}
+
+    <div class="form-actions form-actions__left-on-med">
+        <input class="btn form-actions_item"
+               type="submit"
+               value="Apply Filters">
+        <input class="btn btn__warning btn__link form-actions_item js-form_clear"
+               type="button"
+               value="Clear filters"
+               data-js-hook="form-clear">
+    </div>
+
+</form>
+{% endmacro %}
+
+{# A helper macro for filter_filterable_list.render()
+   ========================================================================== #}
+{% macro _render_form_wrapper(inputs) -%}
+
+    {% for input in inputs recursive %}
+        <div class="form-l_col
+                    form-l_col-{{ input.cols if input.cols else '1' }}">
+            <div class="form-group">
+
+                {% if input.label %}
+                <label class="form-label-header">
+                    {{ input.label }}
+                </label>
+                {% endif %}
+
+                {% if input.type == 'form' %}
+                    <div class="form-l">
+                        {{ loop(input.children) }}
+                    </div>
+                {% elif input.type == 'email'
+                   or input.type == 'number'
+                   or input.type == 'password'
+                   or input.type == 'tel'
+                   or input.type == 'text'
+                   or input.type == 'url' %}
+
+                    <input type="{{ input.type }}"
+                           value="{{ input.attrs.value if input.attrs.value else '' }}"
+                           placeholder="{{ input.attrs.placeholder if input.attrs.placeholder else '' }}">
+                {% elif input.type == 'checkbox' %}
+                    {% for checkbox in input.attrs -%}
+                        <label class="form-group_item">
+                            {{ _filter_checkbox(checkbox.label, checkbox.name, checkbox.value) | safe }}
+                        </label>
+                    {% endfor %}
+                {% elif input.type == 'select' %}
+                    <label class="form-group_item">
+                        <select>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/8a29f17afdc9cddf99b8fc0a55b3d29660cb3f11